spec/lib/generators/good_job/install_generator_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+require 'generators/good_job/install_generator'
+
+describe GoodJob::InstallGenerator, type: :generator do
+  it 'creates a migration for good_jobs table' do
+    create_dummy_app
+
+    install_good_job
+
+    expect(Dir.glob("#{dummy_app_path}/db/migrate/[0-9]*_create_good_jobs.rb")).not_to be_empty
+
+    remove_dummy_app
```

Comment:
If this is moved into an `after { remove_dummy_app }` I think it will cleanup even if the assertion fails.

Reply:
Good point.